src/pages/example--hello-world.mdx

```diff
@@ -274,6 +249,48 @@ async function lock(lovelace, { into, owner }): Promise<TxHash> {
 }
 ```
 
+### Publishing The Locking Contract
+
+Run the excerpt above by executing:
+
+```
+deno run --allow-net --allow-read --allow-env hello_world-lock.ts
+```
+
+The above code requires you to:
+
+- have a `BLOCKFROST_API_KEY` environment variable set. You can get one by [signing up for a Blockfrost account](https://blockfrost.io/).
+
+- have this file (`hello_world-lock.ts`) is placed at the root of your `hello_world` folder.
```

Comment:
This sentence kinda reads odd. Perhaps there is a typo somewhere here.

Reply:
removing the `is` would do it